fdbserver/StorageMetrics.actor.h

```diff
@@ -350,10 +350,11 @@ struct StorageServerMetrics {
 
 		if (sb.free < 1e9 && deterministicRandom()->random01() < 0.1) {
 			TraceEvent(SevWarn, "PhysicalDiskMetrics")
-				.detail("Free", sb.free)
-				.detail("Total", sb.total)
-				.detail("Available", sb.available)
-				.detail("Load", rep.load.bytes);
+			    .suppressFor(60.0)
```

Comment:
I think with this suppression, we can remove the random sampling part of the if block.